proxy/src/control/destination/background.rs

```diff
@@ -79,13 +78,13 @@ impl Config {
     }
 
     /// Bind this handle to start talking to the controller API.
-    pub fn process<T>(self, executor: &Handle) -> Process<T>
```

Comment:
This means we can remove the `Config` and `Process` types (basically, remove the `Config` type and rename the `Process` type to `Background`)!

Reply:
Yeah, I thought that kind of refactor was slightly out of scope for this PR, I'll open an issue for it.

Reply:
Opened  #966 for this.

Reply:
I made this change in a follow-up PR #967